src/work/ZeynepAbes.js

```diff
@@ -36,14 +36,14 @@ export default function ({slug, name}) {
             <ImageCaption
               materials="Photogrammetry of Istiklal Street"
               style={{ minHeight: CAPTION_HEIGHT }}
-             />
+            />
             <p>
-              My home has felt like a different place every time I visit. I shared
```

Comment:
justifying, if zeynep does indeed want it, needs to happen with css, the whitespace from html markdown is not preserved. you can add `className="justify"` into the ProjectColumns tags for these, and then add this entry to `src/Containers/App/style.css`
```css
.justify {	
  text-align: justify !important;	
}
```